src/main/java/com/minecolonies/coremod/colony/buildings/views/AbstractBuildingView.java

```diff
@@ -153,13 +155,21 @@ public boolean isRepairing()
 
     /**
      * Open the associated BlockOut window for this building.
+     * @param sneaking if the player is sneaking.
      */
-    public void openGui()
+    public void openGui(final boolean sneaking)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Provide multiple methods instead of using "sneaking" to determine which action to take. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2301)

Reply:
rename to meaning, not value